UM/View/GL/OpenGL.py

```diff
@@ -98,13 +98,19 @@ def __init__(self) -> None:
 
         self._opengl_version = self._gl.glGetString(self._gl.GL_VERSION) #type: str
 
+        try:
+            self._opengl_shading_language_version = float(self._gl.glGetString(self._gl.GL_SHADING_LANGUAGE_VERSION)[0:3])
+        except:
+            self._opengl_shading_language_version = 1.0
```

Comment:
I would use this as `UM.Version` object which you can use for version comparison. If later the version includes one more part, it won't work with `float`.